### PR TITLE
Fish factory create stop button for conveyor belt

### PR DESCRIPTION
**Pull request by jonp-h**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Adds GameManager to Bleeding Station
Implements stop button

* **What is the current behavior?** (You can also link to an open issue here)
Fish factory does not have GameManager
Fish factory conveyorbelts cannot be stopped
#280 

* **What is the new behavior?** (if this is a feature change)
Adds stop button

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

* **Other information**:
Introduces odd bug: CollisionStay does not run when fish is not moving on stopped conveyor belt.
Solution: Add force to fish when conveyor is started after being stopped.
Refs: FactoryFishSpawner.cs line: 68

Closes: #280

### CODE REVIEW

**Review by krisgjol — APPROVED** [review]

**Review by Herman1412 — APPROVED** [review]
Looks good

File: Assets/FishFactory/Scripts/Common/ConveyorController.cs [hunk 1]
```diff
@@ -12,16 +12,29 @@ private enum Direction
         Left,
     }
 
+    // test with serializedfield //FIXME: remove
+    [SerializeField]
+    private GameManager gameManager;
+
     // Editor fields
     [SerializeField]
```
Comment (MarkusJohansen) [inline Assets/FishFactory/Scripts/Common/ConveyorController.cs:20]: Why is the variable reinitialized as a public variable? 😅

**Review by MarkusJohansen — COMMENTED** [review]
Looks good! Just have a look at the changes proposed, and resolve comments before merging!🚀

File: Assets/FishFactory/Scripts/Common/ConveyorController.cs [hunk 2]
```diff
@@ -12,16 +12,29 @@ private enum Direction
...
     private bool isBeltOn = true;
-    public bool IsBeltOn { get { return isBeltOn; } }
+    public bool IsBeltOn
+    {
+        get { return isBeltOn; }
+    }
+
```
Comment (MarkusJohansen) [inline Assets/FishFactory/Scripts/Common/ConveyorController.cs:26]: Why is the variable reinitialized as a public variable? 😅

**Pull request merged**